install/gains.csv

```diff
@@ -1,6 +1,8 @@
 Station,Location,Sublocation,Subsource,Scale Factor,Scale Bias,Absolute Bias,Start Date,End Date
 ALS1R,01,01,,0.2,,,2023-02-24T01:09:00Z,9999-01-01T00:00:00Z
 ALS2R,01,01,,0.2,,,2023-02-23T03:59:00Z,9999-01-01T00:00:00Z
+EYWM,50,,X,,37683.0 * 0x8f / 0xff,,2013-12-15T00:00:00Z,9999-01-01T00:00:00Z
+EYWM,50,,Z,,-37618.0 * 0x6d / 0xff,,2013-12-15T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
Yes, this is a pain. without adding a bias factor to the calibration code the gain file will still be needed.

Your change isn't right as you've removed the bias factor rather than the gain.

(gain is relative to the nominal gain and bias,)

You would need to change to:

```
EYWM,50,,X,37683.0/37600.0,37683.0 * 0x8f / 0xff,,2013-12-15T00:00:00Z,9999-01-01T00:00:00Z
EYWM,50,,Z,-37618.0/37600.0,-37618.0 * 0x6d / 0xff,,2013-12-15T00:00:00Z,9999-01-01T00:00:00Z
```